install_config/cluster_metrics.adoc

```diff
@@ -19,18 +19,24 @@ exposes metrics that can be collected and stored in back-ends by
 link:https://github.com/kubernetes/heapster[Heapster].
 
 As an {product-title} administrator, you can view a cluster's metrics from all
-containers and components in one user interface. These metrics are also used by
-xref:../dev_guide/pod_autoscaling.adoc#dev-guide-pod-autoscaling[horizontal pod
-autoscalers] in order to determine when and how to scale.
+containers and components in one user interface.
+
+[NOTE]
+====
+Previous verions of {product-title} used metrics from Heapster to configure
+horizontal pod autoscalers. Now horizontal pod autoscalers use metrics from the
+{product-title} metrics server.
+See
+xref:../dev_guide/pod_autoscaling.html#req-for-using-hpas[Requirements for Using Horizontal Pod Autoscalers] for detailed
```

Comment:
When using xrefs you should reference the `.adoc` file instead of `.html`. As is, this renders as `.../dev_guide/pod_autoscaling.html.html#req-for-using-hpas`.

Adjust the xref to be as follows

> xref:../dev_guide/pod_autoscaling.adoc#req-for-using-hpas

Reply:
Updated.